packages/api/src/web5.ts

```diff
@@ -234,9 +234,12 @@ export class Web5 {
         );
       }
 
+      // Use the specified DWN endpoints or the latest TBD hosted DWN
+      const serviceEndpointNodes = techPreview?.dwnEndpoints ?? didCreateOptions?.dwnEndpoints ?? ['https://dwn.tbddev.org/beta'];
```

Comment:
There are two other places further down the flow that get the array of service endpoints in a similar same way, delete the other two in favor of just this one up here.

Reply:
Sounds good

Reply:
Done